timeliney.typ

```diff
@@ -283,17 +284,20 @@
 
                       if show-grid == true or show-grid == "y" {
                         for (i, task) in flat_tasks.enumerate() {
-                          let (_, task_bottom_y, _) = coordinate.resolve(ctx, "titles.task" + str(i) + "-bottom")
+                          let task_bottom_y = coordinate.resolve(ctx, "titles.task" + str(i) + "-bottom").at(1).at(1)
                           line((start_x, task_bottom_y), (end_x, task_bottom_y), ..grid-style)
                         }
 
                         if milestone-layout == "aligned" {
                           for (i, milestone) in milestones.enumerate() {
-                            let (_, bottom_y, _) = coordinate.resolve(ctx, "titles.milestone" + str(i) + "-bottom")
+                            let bottom_y = coordinate.resolve(ctx, "titles.milestone" + str(i) + "-bottom").at(1).at(1)
                             line((start_x, bottom_y), (end_x, bottom_y), ..grid-style)
                           }
                         }
                       }
+
+                      // Border all around the timeline
+                      rect("titles.north-west", (end_x, end_y), stroke: black + 1pt)
```

Comment:
I also added a rectangle around the timeline to have a border even if there is no task group

Reply:
Hey, thank you so much for this PR! I had tried to port it to a new version of CeTZ and couldn't make sense out of my own messy code to find where things were breaking, so thank you for doing that work (and sorry for the mess!)

I am currently on vacation away from home, so I'll only review this on the weekend when I get back home, just wanted to leave this comment so you know I saw this.